product-service-ts/handlers/pgClient.ts

```diff
@@ -0,0 +1,50 @@
+import { Client } from 'pg';
+import { connectionOptions } from './helpers';
+
+export const init = async () => {
+  const client = new Client(connectionOptions);
+  await client.connect();
+
+  try {
+    await client.query(`create extension if not exists "uuid-ossp"`);
+    await client.query(`
+      create table if not exists products(
+        id UUID NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+        title TEXT NOT NULL,
+        description TEXT,
+        price INTEGER
+      )
+    `);
+    await client.query(`
+      create table if not exists stocks(
+        product_id UUID NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+        count integer,
+        foreign key (product_id) references products (id)
+      )
+    `);
+    await client.query(`
+      insert into products (description, flavor, price) values 
+        ('Protein Cookie', 'Coconut', 90),
+        ('Protein Cookie', 'Apple', 100),
+        ('Protein Cookie', 'Wildberry', 90),
+        ('Protein Bar', 'Almond', 120),
+        ('Protein Bar', 'Banana', 120),
+        ('Protein Bar', 'Chocolate', 130),
+        ('Protein Bar', 'Coconut', 150)
+    `);
+    await client.query(`
+      insert into stocks (product_id, count) values 
+        ('a35c018a-091e-48d7-b1e3-d6ef9cb6c589', 4),
```

Comment:
I suppose if you try to run this script on a clean DB you will have new id's for products and this query for `stocks` will try to create rows with hardcoded non-existing `product_id` values.